indexing-service/src/main/java/org/apache/druid/indexing/common/task/AbstractTask.java

```diff
@@ -201,7 +201,11 @@ public void cleanUp(TaskToolbox toolbox, @Nullable TaskStatus taskStatus) throws
     // clear any interrupted status to ensure subsequent cleanup proceeds without interruption.
     Thread.interrupted();
```

Comment:
In previous code, if `isStandAloneTask=true`, even this line doesn't gets executed. Would it create any problems now ?

Reply:
good point, i'm just gonna change the logic back to override cleanup, think its cleaner that way